_community/google-groups.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Google Groups - Managing List Membership
```

Comment:
Overall, I think we should really de-emphasize the Google Groups thing on this page. We are really just using them as glorified mailing lists, and so let’s stick to just that one concept.

So, for a title, how about “Manage Your OSG List Memberships” or something?